python/lsst/ip/isr/isrTask.py

```diff
@@ -1187,6 +1187,17 @@ def run(self, ccdExposure, camera=None, bias=None, linearizer=None, crosstalkSou
                 illumMaskedImage is None):
             raise RuntimeError("Must supply an illumcor if config.doIlluminationCorrection=True.")
 
+        minEdgeSuspectMask = 0
+        if self.config.doBrighterFatter is True:
+            # Applying the brighter-fatter correction applies a
+            # convolution to the science image. At the edges this
+            # convolution may not have sufficient valid pixels to
+            # produce a valid correction. Mark pixels within the size
+            # of the brighter-fatter kernel as SUSPECT to warn of this
+            # fact.
+            minEdgeSuspectMask = numpy.max(bfKernel.shape)
```

Comment:
Should this be halved? Ref https://github.com/lsst/ip_isr/blob/master/python/lsst/ip/isr/isrFunctions.py#L799